index.html

```diff
@@ -220,6 +220,66 @@ <h3>Multi-Post Stories</h3>
             </div>
           </section>
 
+          <!--Contact-form-->
+          <section id="contact" class="c-form">
+            <p class="con-us">CONTACT US</p>
```

Comment:
- [ ] Make sure the text **"CONTACT US"** match the styling like on the Figma design such as its **color**, **font-weight**, **font-family**, **line-heigt**, **font-size**,.... Make sure you check all the properties on the Figma template to match your design to it.

- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you use a `hx(e.g.: h2, h3,...)` tag on the text `CONTACT US` instead of the `p`(as the text is a title and not a paragraph) tag for good semantics tags.